### PR TITLE
Laravel Snippet

### PR DESCRIPTION
**Pull request by FiloSpaTeam**
Snippet useful for Laravel Framework (Php).
Contains:
* Resource Controller
* Service Provider for Service
* Service Provider for Repository
* Base Model
* Base Repository
* Base Service
* Facade

### CODE REVIEW

**Comment by SirVer** [comment]
Does laravel have it's own `filetype` we could use? Most php users will probably not be enthused about having more snippets that pollute the namespace.

**Comment by FiloSpaTeam** [comment]
No, vim return php filetype :(
I can do this:
* Create a function that set correct filetype from a user input when triggered FileType event
* Add a function that read into the project tree or something like that, that search for a file called (project.filetype for example) that contains the correct file type for the original filetype (an example: php laravel.php)
* Idk... XD
Otherwise laravel snippets need php_laravel notation...

File: UltiSnips/php_laravel.snippets [hunk 1]
```diff
@@ -0,0 +1,260 @@
+#resource controller
+snippet l_rsc "Laravel resource controller"
```
Comment (SirVer) [inline UltiSnips/php_laravel.snippets]: most (all?) of the snippets should have the "b" option set.

Comment (FiloSpaTeam) [inline UltiSnips/php_laravel.snippets]: Ok now correct all snippets :+1: 
I've not seen this option xD sorry.

**Comment by SirVer** [comment]
Thanks.

**Pull request merged**